File: eco-core/core-proxy/src/main/java/com/willfp/eco/proxy/SkullProxy.java
package com.willfp.eco.proxy;

import com.willfp.eco.core.proxy.AbstractProxy;
import org.bukkit.inventory.meta.SkullMeta;
import org.jetbrains.annotations.NotNull;

public interface SkullProxy extends AbstractProxy {
    void setSkullTexture(@NotNull SkullMeta meta,
                         @NotNull String base64);
}

File: eco-core/core-proxy/src/main/kotlin/com/willfp/eco/proxy/SkullProxy.kt
package com.willfp.eco.proxy

import com.willfp.eco.core.proxy.AbstractProxy
import org.bukkit.inventory.meta.SkullMeta

interface SkullProxy : AbstractProxy {
    fun setSkullTexture(
        meta: SkullMeta,
        base64: String
    )

    fun getSkullTexture(
        meta: SkullMeta
    ): String?
}